src/devices/Directory.Build.props

```diff
@@ -3,6 +3,10 @@
   <Import Project="../../eng/Versions.props" />
   <Import Project="../../eng/Analyzers.props" />
   <PropertyGroup>
+    <!-- Most projects should use this value to build for all possible target frameworks -->
+    <TargetFrameworksBase>net5.0;netcoreapp2.1;netstandard2.0;</TargetFrameworksBase>
+    <!-- If .NET Standard is not supported (and adding support is not useful) use this instead -->
+    <TargetFrameworksReduced>net5.0;netcoreapp2.1</TargetFrameworksReduced>
```

Comment:
NIT: I would define this one first and then have the one that has netstandard2.0 too be based on this one, that way when we move to target something else the update only goes in one line. Also, I think I would remove these to be more like: DeviceBindingsDefaultTFMs and DeviceBindingsNetCoreTFMs